.github/workflows/clinical-visit.yml

```diff
@@ -0,0 +1,50 @@
+name: Clinical Visit Test
+on:
+  #push:
+    #branches: [master]
+  pull_request:
+    branches: [master]
+  #repository_dispatch:
+    #types: [qa]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      firefox:
+        image: selenium/node-firefox:3.141.59-gold
+        env:
+          HUB_HOST: hub
+          HUB_PORT: 4444
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Checkout ui test framework
+        uses: actions/checkout@master
+        with:
+          repository: openmrs/openmrs-contrib-uitestframework
+      - name: Build ui test framework
+        run: mvn clean install -B
+      - name: Checkout distro/uitests
+        uses: actions/checkout@master
+        with:
+          repository: openmrs/openmrs-distro-referenceapplication
+      - name: Build distro/uitests
+        run: mvn clean install -B
+      - name: Checkout qaframework
+        uses: actions/checkout@master
+        with:
+          repository: ${{github.repository}}
+      #- name: run db and web containers
+        #run: |
+          #docker-compose -f docker/docker-compose-refqa.yml up -d
+      #- name: wait for openmrs instance to start
+        #run: while ! nc -z localhost 8080 </dev/null; do sleep 1; done
```

Comment:
Likewise, presumably these should be uncommented or else I don't see how these tests will succeed.